src/components/semantic/presenters/AudiencePresenter.tsx

```diff
@@ -28,6 +28,10 @@ const csStagedExamBoards: Partial<Record<Stage, ExamBoard[]>> = {
     "scotland_advanced_higher": ["sqa"],
 };
 
+function isExamboardArray(arr: any[]): arr is ExamBoard[] {
```

Comment:
Remove need of `any`:
```suggestion
function isExamboardArray(arr: string[]): arr is ExamBoard[] {
```